### PR TITLE
feat: homepage

### PR DESCRIPTION
**Pull request by TheAmlanSaikia**
Feat: homepage, responsive except navbar
download
.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (hiteshverma27) [inline .vscode/settings.json]: remove this before pushing

File: index.html [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+                        <h5>Home</h5>
+                    </a>
+                </li>
+                <li class="navlinks">
+                    <a href="/" class="link navbar-link text-lg">
+                        <h5>Products</h5>
+                    </a>
+                </li>
+            </ul>
+            <!-- <div class="searchbox-wrapper justify-end">
```
Comment (hiteshverma27) [inline index.html]: remove commented code before pushing

**Review by hiteshverma27 — APPROVED** [review]
Looks good

File: Utility.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+:root{
+    --footer-bg-color: ##f7f7f7;
```
Comment (Aditi0505) [inline Utility.css]: Please remove extra #

File: index.html [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+                </li>
+                <li class="navlinks">
+                    <a href="http://" class="link navbar-link text-lg">Bag</a>
+                </li>
+            </ul>
+        </nav>
+        <header class="header-banner mt-1">
+            <section class="banner-holder flex-row justify-center relative">
+                <img src="/Assests/Images/Shoe Banner Picture -1.jpg" alt="Banner" class="banner-image responsive-image"
+                    srcset="">
```
Comment (Aditi0505) [inline index.html]: Remove srcset if not needed.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. Rest looks good to me.

File: index.html (hunk 2 above)
Comment (TheAmlanSaikia) [reply index.html]: Thanks for pointing.

File: Utility.css (hunk 3 above)
Comment (TheAmlanSaikia) [reply Utility.css]: Thanks for pointing!

**Pull request merged**